pages/home.page.js

```diff
@@ -0,0 +1,15 @@
+import BasePage from '../pages/base.page';
+
+class HomePage extends BasePage{
+
+    get welcomeUser(){return $('.active')}
+
+    async getWelcomeUserText(){
```

Comment:
Remove this method, as it's not really needed for validating the text of a welcome user (you can use a wdio specific matcher to retrieve it given a web element)